e2e-tests/tfm/cypress/e2e/journeys/case-amendments/user-flow/8. amendment-details-manual-approval.spec.js

```diff
@@ -9,15 +9,30 @@ import { PIM_USER_1, UNDERWRITER_MANAGER_DECISIONS, BANK1_MAKER1, ADMIN, CURRENC
 import caseDealPage from '../../../pages/caseDealPage';
 
 context('Amendments - Manual approval journey', () => {
-  // If the expiry & commencement date are the same day of the month then we add one to the month
-  // difference for BS (but not for EWCS)
-  // BUT... If the commencement date is end of month and the expiry date isn't then we don't add one
-  //
-  // In these tests, the mock start date is two years ago today & the mock end date is a month today.
-  // Therefore if today is EOM and a month from now is not EOM we need to NOT add one to the tenor
-  const todayIsEndOfMonth = add(new Date(), { days: 1 }).getDate() === 1;
-  const aMonthFromNowIsEndOfMonth = add(new Date(), { months: 1, days: 1 }).getDate() === 1;
-  const facilityTenor = todayIsEndOfMonth && !aMonthFromNowIsEndOfMonth ? '25 months' : '26 months';
+  /**
+   * The deal in the test is a BS deal.
+   *
+   * The tenor is calculated using the SQL here:
+   * https://github.com/UK-Export-Finance/databases-datateam/blob/163d71b0aef58652db5e6ccf29edce2ce3df30ca/MASTER_DATA/dbo/Functions/fn_DATEDIFF_FULLMONTH.sql
+   *
+   * In these tests, the mock start date is two years ago today & the mock end date is a month today.
+   *
+   * So in this case it boils down to the tenor being 25 months except for:
+   * - if expiry and commencement dates are the same date of month then you add one
+   * - if today is end of month and a month from now is also end of month you add one
+   */
+  let facilityTenor = '25 months';
+
+  if (today.date.getDate() === oneMonth.date.getDate()) {
+    facilityTenor = '26 months';
+  }
+
+  const isTodayEndOfMonth = add(today.date, { days: 1 }).getDate() === 1;
+  const isMonthFromNowEndOfMonth = add(today.date, { months: 1, days: 1 }).getDate() === 1;
```

Comment:
Whilst we're here we could fix the 4-yearly issue that we're actually comparing 2 years ago to today plus one month


```suggestion
  const isCoverStartDateEndOfMonth = add(twoYearsAgo.date, { days: 1 }).getDate() === 1;
  const isCoverEndDateEndOfMonth = add(today.date, { months: 1, days: 1 }).getDate() === 1;
```